site/app/templates/UserProfile.twig

```diff
@@ -9,21 +9,23 @@
                 {% if image is not null %}
                     {{ base64_image(image.getImageBase64MaxDimension(250), image.getMimeType(), '%s %s'|format(user.getDisplayedGivenName(), user.getDisplayedFamilyName())) }}
                 {% else %}
-                    <span class="center-img-tag">N/A</span>
+                    <span class="center-img-tag">{{ localize('My_Profile.Not_Available', "N/A") }}</span>
                 {% endif %}
             </div>
             <span class="edit-user-img">
-                <button class="icon" data-testid="upload-photo-button" tabindex="0" onclick="showUpdateProfilePhotoForm()" aria-label="Edit user profile photo" title="Edit user profile photo">
+                <button class="icon" data-testid="upload-photo-button" tabindex="0" onclick="showUpdateProfilePhotoForm()" aria-label="{{ localize('My_Profile.Edit_Profile_Photo', "Edit user profile photo") }}" title="{{ localize('My_Profile.Edit_Profile_Photo', "Edit user profile photo") }}">
                     <i class="fas fa-upload"></i> {{ localize('My_Profile.Upload_profile_photo', "Upload profile photo") }}
```

Comment:
change the attribute : `Upload_profile_photo` to `Upload_Profile_Photo`
else it cannot be accessible

Reply:
Great! Thanks for the review!